src/Moryx.ControlSystem/VisualInstructions/IVisualInstructor.cs

```diff
@@ -14,34 +15,22 @@ public interface IVisualInstructor : IResource
         /// Only display these instructions
         /// Have to be cleared with the <see cref="Clear"/> method
         /// </summary>
-        long Display(string sender, VisualInstruction[] instructions);
+        long Display(ActiveInstruction instruction);
 
         /// <summary>
         /// Only display these instructions
         /// Instruction will automatically cleared after the given time
         /// </summary>
-        void Display(string sender, VisualInstruction[] instructions, int autoClearMs);
```

Comment:
When you are on it, can you proof read the comments once 😅 
In this case "Instruction will be cleared automatically after the given <param>...."